webpack/assets/javascripts/react_app/components/Settings/SettingsActions.js

```diff
@@ -5,22 +5,20 @@ import {
   GET_SETTING_FAILURE,
 } from './SettingsConstants';
 
-export const loadSetting = settingName => dispatch => {
+export const loadSetting = settingName => async dispatch => {
   dispatch({ type: GET_SETTING_REQUEST });
-
-  return API.get(`/api/v2/settings/${settingName}`)
-    .then(({ data }) => {
-      dispatch({
-        type: GET_SETTING_SUCCESS,
-        response: data,
-      });
-    })
-    .catch(result => {
-      dispatch({
-        type: GET_SETTING_FAILURE,
-        result,
-      });
+  try {
+    const { data } = await API.get(`/api/v2/settings/${settingName}`);
+    return dispatch({
+      type: GET_SETTING_SUCCESS,
+      response: data,
+    });
+  } catch (result) {
+    return dispatch({
+      type: GET_SETTING_FAILURE,
+      result,
```

Comment:
```suggestion
      result: error,
```
(To be safe, don't change the key name in case the `result` key is used elsewhere)